content/applications/hr/payroll/payslips.rst

```diff
@@ -0,0 +1,386 @@
+   practice to do so. The :guilabel:`Compute Sheet` button can be clicked without first saving the
+   payslip. Doing so will save the entry *and* compute the sheet.
+
+Once the payslip form is saved, click the :guilabel:`Compute Sheet` button to register all the
+information, and have the :guilabel:`Salary Computation` tab populated. If any modifications need to
+be made, click the :guilabel:`Edit` button, make the desired changes, then click the
+:guilabel:`Recompute Worked Days` button to have the changes reflected in the :guilabel:`Worked
+Days` and :guilabel:`Salary Computation` tabs.
+
+To print the payslip, click the :guilabel:`Print` button. To cancel the payslip, click the
+:guilabel:`Cancel Payslip` button.
+
+Once everything on the payslip form is correct, click the :guilabel:`Create Draft Entry` button to
+create the payslip. The chatter is automatically updated to show the email sent to the employee,
+along with a PDF copy of the payslip.
+
+.. image:: payslips/payslip-chatter.png
+   :align: center
+   :alt: The new payslip is emailed to the employee and the email appears in the chatter.
+
+Next, the payment must be sent to the employee. To do this, click the :guilabel:`Make Payment`
+button. Doing so reveals a pop-up form, in which the desired :guilabel:`Bank Journal` that the
+payment should be made against must be selected from a drop-down menu. Then, click the
+:guilabel:`Confirm` button to confirm the journal, and return to the payslip.
+
+.. image:: payslips/make-payment.png
+   :align: center
+   :alt: Click Make Payment to send the payment to the employee.
+
+If a payment needs to be cancelled or refunded, click the corresponding :guilabel:`Refund` or
+:guilabel:`Cancel Payslip` button, located at the top of the payslip form.
+
+.. important::
+   In order for a payslip to be paid, the employee *must* have a bank account entered in their
+   contact information. If there is no bank information, a payslip cannot be paid, and an error will
+   appear when the :guilabel:`Make Payment` button is clicked. Banking information can be found in
+   the :guilabel:`Private Information` tab on the employee's card. Edit the employee card, and add
+   banking information, if it is missing.
+
+   .. image:: payslips/banking.png
+      :align: center
+      :alt: Banking information can be entered in an employee's card.
+
+All payslips
+============
+
+To view all payslips regardless of status, go to :menuselection:`Payroll app --> Payslips --> All
+Payslips`. Here, all payslips are organized by batch (in a default list view).
+
+Click on the :guilabel:`▶ (arrow)` next to the individual batch name to view all the payslips in
+that particular batch, along with all the payslip details. The number of payslips in the batch is
+written in parenthesis after the batch name. The status for each individual payslip appears on the
+far-right side, indicating if it is in `Draft Mode`, `Waiting`, or if it is `Done`.
```

Comment:
Status needs a guilabel since it is a column header on the UI in the active view
```suggestion
written in parenthesis after the batch name. The :guilabel:`Status` for each individual payslip
appears on the far-right side, indicating if it is in `Draft Mode`, `Waiting`, or if it is `Done`.
```